client/src/i18n/fr/stock.json

```diff
@@ -249,7 +249,8 @@
       "EXIT_DEPOT"     : "Transfer vers un dépot",
       "EXIT_SERVICE"   : "Distribution à un service",
       "EXIT_SERVICES"   : "Distributions aux services",
-      "EXIT_LOSS"      : "Perte de stock"
+      "EXIT_LOSS"      : "Perte de stock",
+      "SUCCESS" : "Mouvement créé avec succès"
```

Comment:
So the French is a little different than the English for this.  Do you want to add "stock" in to make it clearer?

Reply:
Fixed